src/App.js

```diff
@@ -58,17 +58,16 @@ export default class App extends Component {
                 <a
                   className="list-group-item list-group-item-action"
                   href={`https://myanimelist.net/anime/${e.id}`}
-                  target="_blank"
-                  key={i}
+                  target="_blank" rel="noreferrer"
```

Comment:
Also, is this supposed to fix the console warnings?

Reply:
Yes , all warning issues are resolved now